pkg/virt-api/webhooks/mutating-webhook/mutators/vmi-mutator_test.go

```diff
@@ -1213,4 +1213,35 @@ var _ = Describe("VirtualMachineInstance Mutator", func() {
 		Expect(*status.Memory.GuestCurrent).To(Equal(memory))
 		Expect(*status.Memory.GuestRequested).To(Equal(memory))
 	})
+
+	It("should set current guest CPU topology based on Domain spec", func() {
```

Comment:
I noticed you delete this unit-test scenario from the vm_test.go:
```
			When("isn't set in VMI template", func() {
				It("Set default CPU topology in VMI status", func() {
```
but you did not add it here. Would you consider adding it?

Reply:
thanks!